tests/Microsoft.Bot.Builder.Dialogs.Tests/ChoicePromptTests.cs

```diff
@@ -820,25 +850,25 @@ private Action<IActivity> StartsWithValidator(string expected)
         {
             return activity =>
             {
-                Assert.IsInstanceOfType(activity, typeof(IMessageActivity));
+                Assert.IsAssignableFrom<IMessageActivity>(activity);
```

Comment:
Shouldn't we use `IsType<T>`?
```suggestion
                Assert.IsType<IMessageActivity>(activity);
```

Reply:
It recommends in the potential fixes to use IsAssignableFrom, otherwise it shows the xUnit error.
![image](https://user-images.githubusercontent.com/62260472/92516534-bab11f00-f1eb-11ea-97a5-d3a9d9f0e55b.png)

Reply:
Ok, thanks!